tests/FuzzTests.elm

```diff
@@ -43,6 +43,18 @@ constructing =
                         |> withDefault 42
                         |> expectZipper [] 42 []
             ]
+        , describe "#openAll"
+            [ fuzz (list int) "should focus on each element in the input (exactly once, in order)" <|
```

Comment:
This tests a bunch of things at once (output list has the same length as the input list, etc...), I think it's a pretty strong property.

Reply:
Nice!